### PR TITLE
docs: Added what-if.md

### PR DESCRIPTION
**Pull request by HannesVertics**

### CODE REVIEW

File: what-if.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# What if I need to...?
+
+This document describes possible operations that developers may need to perform.
+
+## Integrating a new city
```
Comment (haphut) [inline what-if.md:5]: Also add parts for:

- Adding a new vendor
  - deliver new MQTT credentials to them securely, username and password per environment
  - for RabbitMQ give them read/write access to their own subtree in MQTT topic space for staging and possibly only read access to production. So one vendor should only see their own data.
  - monitor their behaviour in staging first
- A new vehicle gets APC devices
  - nothing needs to be done but that is good to write down
- If journey-matcher load is too high, separate one journey-matcher per municipality by copying the manifest structure of city-specific manifests

File: what-if.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+Adding a new city works mostly by copy-pasting city-specific manifests and by replacing any reference to the city.
+
+### [waltti-apc-deployment](https://github.com/tvv-lippu-ja-maksujarjestelma-oy/waltti-apc-deployment)
+- Copy manifests in `manifests/*/<city>`
+- Update `FEED_MAP` in `manifests/journey-matcher/overlays/<env>/kustomization.yaml`
+- Add pulsar topics to `streamnative/environment/<env>/topics_source.tf` and apply the terraform.
+
```
[the remark this answers is not in the document]
Comment (haphut) [reply what-if.md:13]: In anonymizer the relevant environment variables are `IS_INITIAL_PROFILE_READING_REQUIRED` and `PROFILE_COLLECTION_BASE`.

[the remark this answers is not in the document]
Comment (HannesVertics) [reply what-if.md:16]: By multiple counting systems in one vehicle do you mean by different vendors?

[the remark this answers is not in the document]
Comment (HannesVertics) [reply what-if.md:16]: For example would a vehicle with a counting device for each door (only one vendor) need `ACCEPTED_DEVICE_MAP` configuration?

[the remark this answers is not in the document]
Comment (haphut) [reply what-if.md:16]: The first one is correct. Different vendors in one bus is unusual. One counting system means the set of installed devices in one vehicle.

**Pull request merged**